Library/Homebrew/cmd/migrate.rb

```diff
@@ -0,0 +1,29 @@
+require "migrator"
+require "formula_renames"
+
+module Homebrew
+  def migrate
+    raise FormulaUnspecifiedError if ARGV.named.empty?
+
+    ARGV.named.each do |name|
```

Comment:
If name doesn't include "/" then it use `Formulary#from_rack` which use `factory` and then resolves the formula as a tap one.